health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -333,87 +532,179 @@
+function setLimitOrOffset(
+  limitOrOffsetConfig: any,
+  params: any,
+  requestBody: any
+) {
   if (limitOrOffsetConfig) {
     if (limitOrOffsetConfig?.isInParams) {
-      _.set(params, limitOrOffsetConfig?.keyPath, parseInt(limitOrOffsetConfig?.value));
+      _.set(
+        params,
+        limitOrOffsetConfig?.keyPath,
+        parseInt(limitOrOffsetConfig?.value)
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)